### ISSUE TITLE
Don't force app users to select a primary filter [2]

### ISSUE THREAD

**Issue by Iheanacho2027** [hzi-braunschweig/SORMAS-Project#1210]
The current operation mode of the search functionality of 1.21.0 is that users have to search for cases with respect to the status of investigation( pending, done and discarded), there should be a mode for "All" (in addition to pending, done and discarded), that allows users to search for all cases irrespective of the investigation status. This is very important for users ease of use

@chinedar @foyiri @MateStrysewskeSym

**Comment by foyiri** [comment]
i think so for instance when a user once to update a cases and he or she can only remember the name of the cases he or she should be able to search using the name of the cases

**Comment by MateStrysewskeSym** [comment]
@foyiri That's already possible.

**Comment by hzi-braunschweig** [comment]
@Iheanacho2027 I dont understand your question. Can you reproduce this and send a screenshot? I believe we already have the all option included with the pending, done and discarded. There is a search button that you can search by case name if you know it, or case id or etc

**Comment by MateStrysewskeSym** [comment]
@hzi-braunschweig No, @Iheanacho2027 is right, the app currently forces users to select an investigation status. We'll discuss this in our next sprint review.

**Comment by MartinWahnschaffeSymeda** [comment]
@bernardsilenou will have a call to clarify when and why this is needed

**Comment by Iheanacho2027** [comment]
Alright

**Comment by Iheanacho2027** [comment]
We are still awaiting Bernard's call on this

**Comment by bernardsilenou** [comment]
We discussed with the Nigerian folks and concluded the need for an interface to view all cases. The reason being that for retrospective case search by a DSNO, He/She does not know the right tap to search the case in especially if is was created long ago by some one from the state level. In the current design, he has to search through the pending, investigated and discarded tabs to know if they case exist or not. There are 2 options that I can suggest:
1. Designing the mobile interface as in the web interface and colour  cases by investigation status
2. Add another drawer for all the cases to the current design.

**Comment by MartinWahnschaffeSymeda** [comment]
Looking at the current UI, we don't have any space left to add another filter:

![grafik](https://user-images.githubusercontent.com/23217632/69163110-f4def200-0aed-11ea-97ff-2ce279788dac.png)

I'd suggest to get either rid of DONE or DISCARDED here and replace it with ALL.

@bernardsilenou @hzi-braunschweig What do you think?

**Comment by Iheanacho2027** [comment]
As a suggestion, is it possible to reduce the graphical space each of the filters (pending,done and discarded) occupies on the UI, to accomodate for ALL?

**Comment by MartinWahnschaffeSymeda** [comment]
Good point... :-)

**Comment by Chinedar** [comment]
good suggestion I believe. Thanks @Henry Henry <iheanachodenwigwe@gmail.com>

**Comment by MartinWahnschaffeSymeda** [comment]
- [ ] extend menu component to allow 4 entries per row
- [ ] add "all" filter to cases list

**Comment by MateStrysewskeSym** [comment]
I've decided against the approach above because I feel like, especially on tablets with a low resolution, four entries per row are too many. Also, I've added the "All" filter to all lists because I feel like they are necessary everywhere, and thus went with the more generic approach of simply adding it as another filter which now pushes the last filter to a new row in the case directory (which was already the case in other directories before).

We might tweak this in the future if it's too much of a problem.
